grabbit/src/main/groovy/com/twcable/grabbit/ClientJobStatus.groovy

```diff
@@ -46,12 +49,22 @@ class ClientJobStatus {
         this.jcrNodesWritten = jcrNodesWritten
     }
 
-
-    public static ClientJobStatus get(@Nonnull JobExplorer explorer, @Nonnull Long jobExecutionId) {
+    /**
+     * Queries the status of a job
+     * @param explorer job explorer to query
+     * @param jobExecutionId the ID to search
+     * @throws {@link NonExistentJobException} if job can not be found. {@link IllegalArgumentException} if explorer or jobExecutionId are null
+     * @return {@link ClientJobStatus}
```

Comment:
The function declaration already says it returns a `ClientJobStatus`: What about it? Can it be null? Are there any restrictions on what its state can and can not be coming out of this function? Otherwise it's just duplicative noise...